packages/kbn-plugin-generator/sao_template/template/.kibana-plugin-helpers.json

```diff
@@ -0,0 +1,5 @@
+{
+  <% if (generateApp) { %>
```

Comment:
shouldn't this be testing `generateScss`?

Reply:
It should, but I actually just removed the check altogether as we [only include the file](https://github.com/elastic/kibana/pull/19643/files#diff-16c5d668d5af35abd4aecc172f2f6efcR79) when generating SCSS.